docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/eks-helm.mdx

```diff
@@ -0,0 +1,360 @@
+
+## Considerations
+
+### Architecture
+
+![Camunda 8 Self-Managed AWS Architecture Diagram](./assets/camunda-8-self-managed-architecture-aws.png)
+
+## Usage
+
+### Environment Prerequisites
+
+In order to streamline the execution of the subsequent commands, it is recommended to export multiple environment variables.
+
+The following are the required environment variables with some example values.
+
+```shell
+# Your standard region that you host AWS resources in
+export REGION=eu-central-1
+# Following two environment variables can be skipped if you don't have a domain
+# The domain name that you intend to use
+export DOMAIN_NAME=camunda.example.com
+# The e-mail to register with Let's Encrypt
+export MAIL=admin@camunda.example.com
+# The Ingress Helm Chart version
+export INGRESS_HELM_CHART_VERSION=4.8.3
+# The External DNS Helm Chart version
+export EXTERNAL_DNS_HELM_CHART_VERSION=1.13.1
+# The Cert-Manager Helm Chart version
+export CERT_MANAGER_HELM_CHART_VERSION=1.13.2
+# The Camunda 8 Helm Chart version
+export CAMUNDA_HELM_CHART_VERSION=8.3.3
+```
+
+Additionally, please follow the guide from either [eksctl](#) or from [Terraform](#) to retrieve the following values, which will be required for subsequent steps. <!-- TODO: Link to sources -->
+
+- EXTERNAL_DNS_IRSA_ARN
+- CERT_MANAGER_IRSA_ARN
+- DB_HOST
+- PG_USERNAME
+- PG_PASSWORD
+- DEFAULT_DB_NAME
+- REGION
+
+### DNS Setup
+
+:::info
+If you don't have a domain name, you can skip to [Camunda 8](#camunda-8).
+:::
+
+#### ingress-nginx
+
+Ingress-nginx is an open-source Kubernetes Ingress controller that provides a way to manage external access to services within a Kubernetes cluster. It acts as a reverse proxy and load balancer, routing incoming traffic to the appropriate services based on rules defined in the Ingress resource.
+
+The following will install `ingress-nginx` in the `ingress-nginx` namespace via Helm. For more configuration options, conduct the [Helm chart](https://github.com/kubernetes/ingress-nginx/tree/main/charts/ingress-nginx).
+
+```shell
+helm upgrade --install \
+  ingress-nginx ingress-nginx \
+  --repo https://kubernetes.github.io/ingress-nginx \
+  --version $INGRESS_HELM_CHART_VERSION \
+  --set 'controller.service.annotations.service\.beta\.kubernetes\.io\/aws-load-balancer-backend-protocol=tcp' \
+  --set 'controller.service.annotations.service\.beta\.kubernetes\.io\/aws-load-balancer-cross-zone-load-balancing-enabled=true' \
+  --set 'controller.service.annotations.service\.beta\.kubernetes\.io\/aws-load-balancer-type=nlb' \
+  --namespace ingress-nginx \
+  --create-namespace
+```
+
+#### external-dns
+
+External-dns is a Kubernetes add-on that automates the management of DNS records for external resources, such as Load Balancers or Ingress controllers. It monitors the Kubernetes resources and dynamically updates the DNS provider with the appropriate DNS records.
```

Comment:
Let's link to the project explicitly: https://github.com/kubernetes-sigs/external-dns